spec/rubocop/cop/rspec/factory_bot/create_list_spec.rb

```diff
@@ -55,7 +55,13 @@
 
     it 'ignores n.times with argument' do
       expect_no_offenses(<<~RUBY)
-        3.times { |n| create :user, created_at: n.days.ago }
+        3.times { |n| create :user, position: n }
+      RUBY
+    end
+
+    it 'ignores when a method is called in factory arguments' do
+      expect_no_offenses(<<~RUBY)
+        3.times { create :user, position: rand }
```

Comment:
This is where it gets tricky.
There can be a positional argument, and a hash argument with a method call.

I suggest using child argument matching for that, e.g.:
```
def_node_matcher :has_method_call_in_hash?, '(hash <(pair _ send) ...>)'
def_node_matcher :has_method_call_in_positional_args?, '<send ...>'
```

Reply:
A random example you can use for inspiration https://github.com/rubocop-hq/rubocop-rspec/pull/804/files#diff-321b4c4eae6bca30faa33539f9cfb4accf057592521e9a9eaf6a897001c349a8R42

Reply:
why should we differenciate them if anyway they will both discard a potential offence detection

Reply:
Aha! I've missed that you're using https://docs.rubocop.org/rubocop-ast/node_pattern.html#for-descendants 👍 

All good and I'm happy if the cop detects:
```ruby
        3.times { |n| create :user, :active, position: 123 }
```
and ignores:
```ruby
        3.times { |n| create :user, :active, position: n }
```

Reply:
that's what the specs are describing so we should be good